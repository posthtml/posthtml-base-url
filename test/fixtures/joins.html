<script src="test.js"></script>
<img src="test.jpg">
<img src="../test.jpg">
<img src="../../test.jpg">
