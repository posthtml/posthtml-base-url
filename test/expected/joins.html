<script src="assets/test.js"></script>
<img src="images/test.jpg">
<img src="test.jpg">
<img src="../test.jpg">
